fuse_viz/include/fuse_viz/serialized_graph_display.h

```diff
@@ -46,6 +46,7 @@
 #include <memory>
 #include <string>
 #include <unordered_map>
+#include <map>
```

Comment:
I don't suppose you can move this up above `<memory>`? I try (unsuccessfully) to keep these alphabetized.

Reply:
Done.

BTW I actually use a `vim` plugin that automatically `clang-format`s every C++ file on exit, using the first `.clang_format` file in the path up from the cwd. However, I always have to exit `vim` with `:noautocmd x` so it doesn't run because many things in `fuse` don't follow the style rules in that `.clang_format` file I have.

It'd be great if we could have such a `.clang_format` file in `fuse`. I'm happy to contribute that if you want, and add some `// clang-format off` - `// clang-format on` blocks where needed, since there are some pieces of code that are better formatted as they already are. :smiley: 

I wonder what are your thoughts about this?